### PR TITLE
add missing import

### PR DESCRIPTION
**Pull request by dusvyat**
forgot to add numpy import in recommenders.py
brought over test from mindsb

### CODE REVIEW

File: dataprep_ml/recommenders.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 import scipy as sp
 from pydantic import BaseModel
 
+import numpy as np
```
Comment (dusvyat) [inline dataprep_ml/recommenders.py:7]: doh! forgot to move this import across

File: pyproject.toml [hunk 2]
```diff
@@ -1,24 +1,26 @@
...
 python-dateutil = ">=2.1"
-scipy = ">=1.5.4"
+scipy = ">=1.10.1"
 type-infer = ">=0.0.7"
 dataclasses-json = "^0.5.4"
 colorlog = "^6.5.0"
 pydantic = "^1.10.9"
 
+[tool.poetry.dev-dependencies]
+pytest = "^7.4.0"
```
Comment (dusvyat) [inline pyproject.toml:23]: adding pytest as a dev dependency

File: tests/unit_tests/test_recommenders.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
+from dataprep_ml.recommenders import RecommenderPreprocessor
+import scipy as sp
+
+
+def test_preprocessing_cf(recommender_interaction_data):
```
Comment (dusvyat) [inline tests/unit_tests/test_recommenders.py:5]: moving test across from mindsdb

File: pyproject.toml [hunk 4]
```diff
@@ -13,12 +13,14 @@ numpy = "^1"
 pandas = "^1"
 pydateinfer = "0.3.0"
 python-dateutil = ">=2.1"
-scipy = ">=1.5.4"
+scipy = ">=1.7.3"
```
Comment (paxcema) [inline pyproject.toml:16]: Curious about this: any specific reason for the upgrade?

**Review by paxcema — APPROVED** [review]

File: pyproject.toml (hunk 4 above)
Comment (dusvyat) [reply pyproject.toml:16]: I was having issues getting tests to pass with earlier version, bumping v seemed to fix it

**Pull request merged**